src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -151,3 +156,66 @@ def signature_hash_algorithm(self):
             raise UnsupportedAlgorithm(
                 "Signature algorithm OID:{0} not recognized".format(oid)
             )
+
+    @property
+    def extensions(self):
+        # TODO: remove this when we finish landing support for extensions.
+        warnings.warn(
+            "Extension support is not fully implemented. Unsupported "
+            "extensions are IGNORED at this time."
+        )
```

Comment:
Vectors that were necessary for testing this?

On Fri, Mar 27, 2015 at 9:55 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/backends/openssl/x509.py
> <https://github.com/pyca/cryptography/pull/1769#discussion_r27341397>:
>
> > @@ -151,3 +156,66 @@ def signature_hash_algorithm(self):
> >              raise UnsupportedAlgorithm(
> >                  "Signature algorithm OID:{0} not recognized".format(oid)
> >              )
> > +
> > +    @property
> > +    def extensions(self):
> > +        # TODO: remove this when we finish landing support for extensions.
> > +        warnings.warn(
> > +            "Extension support is not fully implemented. Unsupported "
> > +            "extensions are IGNORED at this time."
> > +        )
>
> That would be very difficult at the moment because several of our vectors
> contain multiple critical extensions that we don't support.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1769/files#r27341397>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084